wiki/en/Installation-for-Linux.md

```diff
@@ -54,6 +54,10 @@ For installers on other distributions, see their package managers and [Repology]
 
 ## Set up your hardware
 
+If your distribution uses JACK as sound server, go to "Configure JACK with QjackCtl".
```

Comment:
Is this something we could detect and automate with the .deb install?

Reply:
The sound server is a choice made by the distribution and/or the user.
We explain that a configuration is needed for JACK or PipeWire.
I don't think we need to go any further.
What's more, I don't know how to do it concretely.

Reply:
A script ?